src/handlers/messagedelete.ts

```diff
@@ -22,55 +22,51 @@ module.exports = {
             limit: 1,
             type: AuditLogEvent.MessageDelete,
         });
-
         const deletionLog = fetchedLogs.entries.first();
-
         const currentDate = new Date();
         const formattedDate: string = moment(currentDate).utcOffset(0).format('DD, MM, YYYY, HH:mm:ss');
-
         const userLogsChannel = message.guild.channels.resolve(Channels.USER_LOGS) as TextChannel | null;
-
-        const messageDeleteEmbedNoLog = (formattedDate) => makeEmbed({
+        const messageEmbeds = message.embeds.length > 0 ? message.embeds : [];
+        const messageComponents = [];
+        if (message.content) {
+            messageComponents.push(message.content);
+        }
+        if (message.attachments) {
+            message.attachments.forEach((attachment) => {
+                if (attachment.url || attachment.proxyURL) {
+                    messageComponents.push(attachment.url ? attachment.url : attachment.proxyURL);
+                }
+            });
+        }
+        for (let i = 0; i < messageEmbeds.length; i++) {
+            const messageEmbed = messageEmbeds[i];
+            const { image, fields } = messageEmbed;
+            if (image) {
+                messageComponents.push(`<${image.url}>`);
+            }
+            for (let j = 0; j < fields.length; j++) {
+                const field = fields[i];
```

Comment:
Same here. And here you probably wanted to use the j instead of the i to get to the fields.
```suggestion
            for (const field of fields) {
```